src/containerapp/azext_containerapp/_params.py

```diff
@@ -424,3 +424,19 @@ def load_arguments(self, _):
         c.argument('workload_profile_type', help="The type of workload profile to update. Run 'az containerapp env workload-profile list-supported -l <region>' to check the options for your region.")
         c.argument('min_nodes', help="The minimum node count for the workload profile")
         c.argument('max_nodes', help="The maximum node count for the workload profile")
+
+    # Patch
+    with self.argument_context('containerapp patch list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('managed_env', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('show_all', help='Show all patchable and unpatchable container apps')
```

Comment:
consider using `action='store_true'` if you only accept the flag form

Reply:
Thanks @Juliehzl! Could you please clarify if `action='store=true'` sets the default value as True? If yes, we probably need to set it to `False` instead.

Reply:
Updated. Set  `action='store_true'`. I might have misunderstood what this `action='store-true'` means. 
Some context around the `show-all` flag -- This flag is `false` by default. If user specifies `show-all` while running a patch command like `az containerapp patch list --show-all`, the value of show-all will be true and would list both patchable and un-patchable container apps. We have logic around checking the value of `show-all` to determine if we want to list both patchable and un-patchable container apps or not.
I was able to verify the show-all flag works as expected after the change but I'm still not sure what it means to add this here.
@Juliehzl could you please clarify just to be sure we're on the same page?